data/json/items/vehicle/engine.json

```diff
@@ -39,6 +39,21 @@
       "fault_engine_starter"
     ]
   },
+  {
+    "abstract": "engine_steam",
+    "copy-from": "engine_combustion",
+    "type": "ENGINE",
+    "name": "base steam engine",
+    "looks_like": "v12_diesel",
+    "faults": [
+      "fault_engine_belt_drive",
+      "fault_engine_filter_air",
+      "fault_engine_filter_fuel",
```

Comment:
I don't know much about steam engines, but is there actually a fuel filter for solid fuel like coal?

Reply:
Probably not!

Reply:
Nonsense faults removed.